ds_judgements_public_ui/templates/forms/widgets/court_multiple_input.html

```diff
@@ -0,0 +1,17 @@
+{% with id=widget.attrs.id %}
+  <div {% if id %}id="{{ id }}"{% endif %}
+       {% if widget.attrs.class %}class="{{ widget.attrs.class }}"{% endif %}>
+    {% for group, options, index in widget.optgroups %}
+      {% if group %}
+        <div class="structured-search__court-group-label">
+          <label>{{ group }}</label>
+        </div>
+        <div class="structured-search__court-group">
+        {% endif %}
```

Comment:
More confused indentation.

Reply:
Annoyingly this is being auto-formatted by the pre-commit!